moha/rauk/rauk.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np 
+
+from scipy import constants 
+
+from scipy.special import gamma
+
+from moha.utils import get_atom_type
+
+from scipy.sparse import csr_matrix, diags
+import os
+import json
+
+def build_one_body_matrix(connectivity, atom_dictionary, atoms_num, n_sites, bond_dictionary):
+    # Populate diagonal and non-diagonal matrix elements
+    param_diag_mtrx = np.zeros((n_sites, n_sites))
+    for atom, site in atoms_num:
+        key = atom
+        param_diag_mtrx[site - 1, site - 1] = atom_dictionary[key]
+    print(param_diag_mtrx)
+
+    param_nodiag_mtrx = np.zeros((n_sites, n_sites))
+    for atom1, atom2, _ in connectivity:
+        atom1_name, site1 = get_atom_type(atom1)
+        atom2_name, site2 = get_atom_type(atom2)
+        key1 = atom1_name #if atom1_coord else atom1_name
+        key2 = atom2_name #if atom2_coord else atom2_name
+
+        #print((site1 - 1, site2 - 1), key1 + key2, self.bond_dictionary[key1 + key2])
+        param_nodiag_mtrx[site1 - 1, site2 - 1] = bond_dictionary[key1 + key2]
+        
+    param_nodiag_mtrx = np.minimum(param_nodiag_mtrx, param_nodiag_mtrx.T)
+    param_nodiag_mtrx = param_nodiag_mtrx
+    param_diag_mtrx = csr_matrix(param_diag_mtrx)
+    param_nodiag_mtrx = csr_matrix(param_nodiag_mtrx)
+    
+    one_body = param_diag_mtrx + param_nodiag_mtrx
+    
+    return one_body
+
+def assign_rauk_parameters(connectivity, atom_types, atoms_num, n_sites):
+    r"""Assigns the alpha and beta value from Rauk's table in matrix form"""
+
+    dir_path = os.path.dirname(os.path.abspath(__file__))
+
+    # Paths to the JSON files
+    hx_dict_path = os.path.join(dir_path, 'hx_dictionary.json')
+    kxy_matrix_path = os.path.join(dir_path, 'kxy_matrix_1.json')
+
+    # Load the dictionary from the hx_dictionary.json file
+    with open(hx_dict_path, 'r') as file:
+        hx_dictionary = json.load(file)
+
+    # Load the matrix from the kxy_matrix_1.json file
+    with open(kxy_matrix_path, 'r') as file:
+        kxy_matrix_1_list = json.load(file)
+
+    # Convert list back to numpy array
+    kxy_matrix_1 = np.array(kxy_matrix_1_list)
+    kxy_matrix = np.minimum(kxy_matrix_1, kxy_matrix_1.T) #Symmetric matrix
+
+    alphaC = -0.414 #Value for sp2 orbital of Carbon atom.
```

Comment:
to be consistent with the pep8 style it's better to choose the name as `alpha_c`